core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/StudyImpl.java

```diff
@@ -344,4 +311,21 @@ public ChangeLog createChangeLog(String summary, String columnsChanged, User use
     changeLog.setUser(user);
     return changeLog;
   }
+
+  @Override
+  public String getDeleteType() {
+    return "Study";
+  }
+
+  @Override
+  public String getDeleteDescription() {
+    Project p = getProject();
+    return (p.getShortName() == null ? p.getAlias() : p.getShortName())
+        + " " + getName() + " (" + getAlias() + ")";
+  }
+
+  @Override
+  public SecurityProfile getDeletionSecurityProfile() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getSecurityProfile" on line 205. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)